deploy/configs/serial-libraries/modules.yaml

```diff
@@ -33,6 +33,7 @@ modules:
       - highfive~mpi
       - neuron
       - python-dev
+      - snakemake
```

Comment:
Am I missing where this deployed?  I don't see it at the moment.

Reply:
Argh. I mismatched stuff.

Reply:
NP, was just wondering if I wasn't looking the right place.  Let me know when it's there.

Reply:
Will do

Reply:
@mgeplf good to go now!

Reply:
Cool, I will try and run your building review later today :crossed_fingers:.

BTW: is there a way to have GitHub show the whole file for context? Or does Gitlab?  I like that feature in gerrit, because it allows me to catch 'scoping' mistakes, like what caused this one.